cli/main.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	cli "github.com/urfave/cli/v2"
+)
+
+func main() {
+	app := &cli.App{
+		Name:     "dyo",
+		Version:  "0.1.1",
+		HelpName: "dyo",
+		Usage:    "dyo - cli tool for deploying a complete Dyrectorio stack locally, for demonstration, testing, or development purposes",
+
+		Commands: []*cli.Command{
+			{
+				Name:    "init",
+				Aliases: []string{"i", "g", "gen", "generate"},
```

Comment:
I think use one word word one command its better, but talk about this in person.

Reply:
As per synonyms, "up" would be nice too, but we don't need so many. Generate is more appropriate, if there is i why there is no install pair for it? We can have it consistent with gen only.

up: could do gen and up in sequence, in another command.

Reply:
we talked about it, fixed